README.md

```diff
@@ -59,75 +59,146 @@ It is possible to change the behaviour of the data builder by defining environme
 
-Runs routing quality test bench defined in the repository 'hsldevcom/OTPQA'. OTPQA test sets are associated with GTFS packages.
-If there are quality regressions, a comma separated list of failed GTFS feed identifiers is is written to local file 'failed_feeds.txt'.
+Builds a new street only graph with all the new data sets (and maybe seeded data sets if there were issues with new data).
 
-9. `router:store`
+- `router:buildWithPrebuiltStreetGraph`
+  - `router:copyForPrebuiltStreetGraphDataBuild`
+  - `buildOTPGraphTask(config.router)`
+
+Builds a new graph from prebuilt street only data with new gtfs data sets (and maybe seeded data sets if there were issues with new data).
+
+- `test.sh`
+
+Runs routing quality test bench defined in the repository `hsldevcom/OTPQA`. OTPQA test sets are associated with GTFS packages.
+If there are quality regressions, a comma separated list of failed GTFS feed identifiers is is written to local file `failed_feeds.txt`.
+
+- `router:store`
 
 Stores the new data in storage (which can be a mounted storage volume).
 
-10. `deploy.sh`
+- `router:storeForPrebuiltStreetGraphDataBuild`
+
+Stores the new data in storage (which can be a mounted storage volume). Also copies the `report` directory from the street only build to the output directory under the name `osm-report`.
 
-Deploys new opentripplanner-data-server image with 'DOCKER_TAG' env variable (default 'v3') postfixed with the router name and
+- `deploy.sh`
+
+Deploys a new opentripplanner-data-server image with the `DOCKER_TAG` env variable (default `v3`) postfixed with the router name, and
 pushes the image to Dockerhub.
 
-Normally, when the application is running as a container, the script 'index.js' is run to execute all steps 1 - 10 described above.
-The end result of the build is a data server image uploaded into dockerhub.
+Normally, when the application is running as a container, the script `index.js` is run to execute all steps.
+The end result of the build is a data server image uploaded to dockerhub.
 
-Each data server image runs a http server listening to port 8080, serving both a data bundle required for building a graph,
-and a pre-built graph. For example, in HSL case: http://localhost:8080/router-hsl.zip and graph-hsl-$OTPVERSION.zip. The image
+Each data server image runs a http server listening to port `8080`, serving both a data bundle required for building a graph,
+and a pre-built graph. For example, in the HSL case: http://localhost:8080/router-hsl.zip and `graph-hsl-$OTPVERSION.zip`. The image
 does not include the data, the data needs to be mounted while running the container.
 
-11. `deploy-otp.sh`
+- `deploy-otp.sh`
 
-Tags opentripplanner image with using 'OTP_TAG' env variable (default 'v3') postfixed with the router name and pushes the image to Dockerhub.
+Tags an OTP image using the `OTP_TAG` env variable (default `v3`) postfixed with the router name and pushes the image to Dockerhub.
```

Comment:
Default is `v2`, looks like I've made a mistake before.

Reply:
I changed v3 to v2 in this readme